framework/scripts/wazuh_server.py

```diff
@@ -37,6 +37,10 @@
 MANAGEMENT_API_DAEMON_NAME = 'wazuh-apid'
 
 
+class DaemonStartError(Exception):
```

Comment:
We should move this exception to https://github.com/wazuh/wazuh/blob/master/framework/wazuh/core/exception.py, it could be a child of `InternalServerError`